abnf_to_regexp/single_regexp.py

```diff
@@ -93,7 +95,7 @@ def convert_repetition(self, element: Repetition) -> str:
             and element.max_occurrences is not None
             and element.max_occurrences > 0
         ):
-            suffix = f"{{{element.max_occurrences}}}"
+            suffix = f"{{,{element.max_occurrences}}}"
```

Comment:
A small note here, this could also be written as `suffix = f"{{0,{element.max_occurrences}}}"` which might be better for readability.

Reply:
@oliverhaagh could you please make a commit? I'll squash them together then before merge.

Reply:
Yes, I will do that. :+1: